pkg/config/planargument.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+import (
+	"github.com/lunarway/shuttle/pkg/git"
+	"os"
+	"path"
+	"strings"
+)
+
+func isPlanArgumentAFilePlan(planArgument string) bool {
+	return strings.Index(planArgument, "/") == 0 || strings.Index(planArgument, "./") == 0 || strings.Index(planArgument, "../") == 0
+}
+
+func isPlanArgumentAPlan(planArgument string) bool {
+	return planArgument != "" && (git.IsGitPlan(planArgument) || isPlanArgumentAFilePlan(planArgument))
+}
+
+func getPlanFromPlanArgument(planArgument string) string {
+	switch {
+	case isPlanArgumentAFilePlan(planArgument) && isFilePath(planArgument, true):
+		return planArgument
+	case isPlanArgumentAFilePlan(planArgument) && isFilePath(planArgument, false):
+		wd, err := os.Getwd()
+		if err != nil {
+			panic(err)
```

Comment:
This ain't nice. Our error handling is in general not good and I would prefer this to return an error instead as it makes future improvements easier. My point is, new functions should make the error handling up to the caller instead.

Do you agree? If not just leave it and we can do an error handling overhaul in the near future.

Reply:
I would agree with you, except in this case. This is extremely unlikely to fail, which is why I didn't want to expand the api with an error.